ephios/templates/base.html

```diff
@@ -116,12 +121,17 @@
                                     {% endwith %}
                                 </ul>
                                 <ul class="navbar-nav ms-auto">
+                                    {{ signalled_nav_html|safe }}
                                     <li class="nav-item dropdown">
                                         <a class="nav-link dropdown-toggle" href="#" id="navbarDropdownAccount" role="button"
                                            data-bs-toggle="dropdown" aria-expanded="false">
                                             {{ user.get_full_name }}
                                         </a>
-                                        <div class="dropdown-menu" aria-labelledby="navbarDropdownAccount">
+                                        <div class="dropdown-menu dropdown-menu-end" aria-labelledby="navbarDropdownAccount">
+                                            {% for item in nav_userprofile %}
+                                                <a class="dropdown-item" {% include "core/fragments/attrs.html" with item=item %}
+                                                   href="{{ item.url }}">{{ item.label }}</a>
+                                            {% endfor %}
                                             <a class="dropdown-item"
                                                href="{% url "core:workinghours_own" %}">{% translate "Working hours" %}</a>
```

Comment:
is there a reason you left those in the HTML instead of moving it to the python code like with the sections?

Reply:
I thought it would be more work to move it. For the sections it make sense because we could automate rendering the section headings.